daemon/indexing/apollo/index.js

```diff
@@ -201,7 +201,7 @@ const typeDefs = gql`
 
   # The "Query" type is the root of all GraphQL queries.
   type Query {
-    listings(searchQuery: String, filters: [ListingFilter!]): ListingPage,
+    listings(searchQuery: String, filters: [ListingFilter!], itemsPerPage: Int = 10, fromPage: Int = 1): ListingPage,
```

Comment:
How about using the Page object as input rather than 2 separate fields ?

Reply:
yup sounds good 👍